lib/fastlane/actions/get_build_number.rb

```diff
@@ -7,7 +7,7 @@ def self.run(params)
         # More information about how to set up your project and how it works:
         # https://developer.apple.com/library/ios/qa/qa1827/_index.html
 
-        folder = params[:xcodeproj] ? File.join('.', params[:xcodeproj], '..') : '.'
+        folder = params[:xcodeproj] ? File.join(params[:xcodeproj], '..') : '.'
```

Comment:
Haven't thought about using that, you could take over and make a new PR, this one is already merged.